opmon/firefox-desktop-networking.toml

```diff
@@ -0,0 +1,73 @@
+[project]
+
+name = "Firefox Desktop Networking Monitoring"
+platform = "firefox_desktop"
+xaxis = "submission_date"
+start_date = "2024-12-05"
+# We want continuous monitoring of this data
+#end_date =
+skip_default_metrics = true
+
+metrics = [
+  "total_network_close_errors"
+]
+
+alerts = [
+  "historical_diff_total_network_close_errors"
+]
+
+
+[project.population]
+
+data_source = "network_close_reasons"
+monitor_entire_population = true
+dimensions = ["network_close_reason"]
+group_by_dimension = "network_close_reason"
```

Comment:
This will create separate graphs and alerts for every error type/key value in `networking_http_3_connection_close_reason`